vividus-bdd-engine/src/main/java/org/vividus/bdd/output/model/TestCaseType.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.bdd.output.model;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import org.vividus.bdd.model.jbehave.Scenario;
+import org.vividus.bdd.model.jbehave.Step;
+
+public enum TestCaseType
+{
+    MANUAL, AUTOMATED;
+
+    public static TestCaseType getTestCaseType(Scenario scenario)
+    {
+        return getTestCaseType(scenario.collectSteps());
+    }
+
+    public static TestCaseType getTestCaseType(List<Step> steps)
+    {
+        return steps.stream().allMatch(isManual()) ? MANUAL : AUTOMATED;
+    }
```

Comment:
it should be moved to `Scenario`

Reply:
okay, but don't like this general idea of mixing up test case management notions and jbehave notions, it makes it impossible to move these models and methods into jbehave since there are no such things as test cases, types, test suites etc...